MediaGalleryUi/view/adminhtml/web/template/grid/columns/image/actions.html

```diff
@@ -5,9 +5,9 @@
  */
 -->
 
-<!-- ko  foreach: { data: actionsList, as: 'action' } -->
+<each args="{ data: actionsList, as: 'action' }">
     <li>
         <a class="action-menu-item" href=""
            data-bind="text: action.title, click: $parent[action.handler].bind($parent, $row()), 'data-action': 'item-' + action.name" />
```

Comment:
Can you please check if remaining bindings can also be updated?
```suggestion
           text="action.title" click="$parent[action.handler].bind($parent, $row())" attr="'data-action': 'item-' + action.name" />
```

Reply:
Thanks for the review @sivaschenko ! Bindings is now updated. Although this particular template seems to be overriden by AdobeStockImageAdminUi/view/adminhtml/web/template/mediaGallery/grid/columns/image/licenseActions.html , I have updated it as well.